docs/source/delta-clustering.md

```diff
@@ -34,7 +34,8 @@ To enable liquid clustering, add the `CLUSTER BY` phrase to a table creation sta
   CREATE TABLE table1(col0 int, col1 string) USING DELTA CLUSTER BY (col0);
 
   -- Using a CTAS statement
-  CREATE TABLE table2 CLUSTER BY (col0)  -- specify clustering after table name, not in subquery
+  CREATE EXTERNAL TABLE table2 CLUSTER BY (col0)  -- specify clustering after table name, not in subquery
+  LOCATION 'table_location'
```

Comment:
do we need location here, previously it was not there

Reply:
yes, we now supports creating table from external location in https://github.com/delta-io/delta/pull/3251